scripts/mitochondria_m2_wdl/AlignAndCall.wdl

```diff
@@ -426,19 +427,18 @@ task M2 {
     Boolean? make_bamout
     Boolean compress
     File? gatk_override
+   
     # runtime
     String? gatk_docker_override
-    Int? mem
```

Comment:
I think the `mem`, `command_mem`, and `machine_mem` arguments for this task were already hooked up correctly. Since mem was an optional input you could override the memory for this task and the java command appropriately the way it was set up before.